wordpress-mobile/WordPress-Android
# Fix dark mode text color for new follow button, interest chip

This PR updates `colorOnSecondary` [from black to white in the dark mode](https://github.com/wordpress-mobile/WordPress-Android/blob/974f5f219ddc2c0680f48ce89e03ea7c360b518a/WordPress/src/main/res/values-night/styles.xml#L13). `colorSecondary` is a pink shade in both light + dark mode and black `colorOnSecondary` didn't look good on it.

cc @mbshakti 

`colorOnSecondary`  is used in the new follow button ([Figma design link](https://www.figma.com/file/IDwOB0lUVRDCVAQ80YNNCz/%F0%9F%93%90-Topic-Detail-Page?node-id=194%3A468)) and interest chip text color selectors. It is not used anywhere else in the app.

To test:

Prerequisites - Reader Improvements feature flag is enabled 

- Launch the app and switch to dark mode
- Go to Reader tab
- Notice that interest picker is shown 
- Click an interest chip
- Notice that selected chip text color is white

-------

- Launch the app and switch to dark mode
- Go to Reader => Topic/ Tag details page
- Notice that Follow button text color is white

|Before|After| 
|---|---|
| ![food_before](https://user-images.githubusercontent.com/1405144/87627180-c0d6d200-c74b-11ea-976f-2c083eb7ab06.png) | ![food_new](https://user-images.githubusercontent.com/1405144/87626800-e44d4d00-c74a-11ea-9c8d-9e4a4afcb90e.png)| 
| ![interests_before](https://user-images.githubusercontent.com/1405144/87627187-c6ccb300-c74b-11ea-96f8-93afd920be56.png) | ![inetersts](https://user-images.githubusercontent.com/1405144/87626816-efa07880-c74a-11ea-8efc-d8e9f6354b53.png)|

**Merge instructions**

- Review this PR
- Make sure #12478 is merged
- Update target branch to develop
- Remove the "Not ready for merge" label
- Merge this PR

PR submission checklist:
